README.md

```diff
@@ -18,6 +21,27 @@ We write about everything that happens to the PrestaShop software:
 
 This devblog also hosts How-Tos on certain features, before they find their way in the Developer Guide or the Designer Guide.
 
+## Run locally
+
+build.prestashop.com is hosted by GitHub Pages and generated by [Jekyll](https://jekyllrb.com/) which is a Ruby static site generator.
```

Comment:
it's generated by `github-pages` https://github.com/PrestaShop/prestashop.github.io/blob/master/Gemfile#L2
Yes it used Jekyll but the main package is https://github.com/github/pages-gem

Reply:
```suggestion
build.prestashop.com is built and hosted by [GitHub Pages](https://github.com/github/pages-gem) which leverages [Jekyll](https://jekyllrb.com/) (a Ruby static site generator) abilities.
```